pkg/cmd/workspace/start.go

```diff
@@ -96,3 +106,72 @@ func startAllWorkspaces() error {
 	}
 	return nil
 }
+
+func compFlagCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
I think we should change this function name to something like getProjectNameCompletions for more clarity